docs/content/en/releases/0.15.0.md

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
@lucperkins I'm curious if we could avoid the need for creating these release md files in the future given the cue data or if these is something specific to the framework that makes this difficult.

Reply:
This isn't possible without a script, as Hugo doesn't create files. Though such a script is certainly feasible, I think we should strive to avoid complicating factors like that as much as possible.

Reply:
Gotcha, yeah, it seems like we could script around it but agreed that it isn't a big deal for releases given their frequency.